src/workflows/publish.sh

```diff
@@ -30,20 +30,6 @@ if package_exists "$package_name" && grep --silent --line-regexp --fixed-strings
   exit 1
 fi
 
-version_tag="v$version"
-if git fetch --end-of-options origin "refs/tags/$version_tag" 2>/dev/null; then
-  echo "Tag $version_tag is already present"
-  exit 1
-fi
```

Comment:
Given that the tag must be created, I would keep this check, modifying it to:

1. Make sure the tag exists (should always be true since that's the new trigger for this workflow)
2. Check the newly created tag matches the package.json's version

Reply:
added these two cases